README.md

```diff
@@ -28,11 +28,12 @@ Setup for developers
   shell variable every time you activate the virtualenv, edit `venv/bin/activate`
   and add to the bottom the export statement.
 7. Run `python systers_portal/manage.py migrate`.
-8. Run `python systers_portal/manage.py runserver` to start the development server. When in testing
+8. Run `python systers_portal/manage.py createsuperuser`.You will be prompted to enter your username for the superuser followed by e-mail and password.Reconfirm the password.You will be prompted with the message "Superuser created successfully".
```

Comment:
Don't ignore the common typography rules. Use space after punctuation marks (dot, comma, etc).

A better approach would be to say "Run _this_ to do _that_", rather than describing the whole prompt process. I would suggest to have something like "Run _this_ to be able to login to admin panel."

The process description in this case is not relevant, since this is the only possible scenario and normally any user will follow the instructions. There is no way you can mess up this :smile: